arch/arm/boot/dts/overlays/rpi-poe-plus-overlay.dts

```diff
@@ -16,6 +16,21 @@
 			};
 		};
 	};
+	fragment@11 {
+		target = <&poe_mfd>;
+		__dormant__ {
+			rpi-poe-power-supply@f2 {
+				compatible = "raspberrypi,rpi-poe-power-supply";
+				reg = <0xf0>;
```

Comment:
Sorry - I just noticed this should be 0xf2.

Reply:
Agreed, but then needs a slight rejig as RPI_POE_ADC_REG is currently 2, so that needs to be reset to 0, and FW needs a base offset to be added.

Reply:
Done.